modules/user-groups/user-groups.php

```diff
@@ -1,573 +1,613 @@
+			}
+
+			$args = array(
+				'message' => 'usergroup-updated',
+			);
+			$redirect_url = $this->get_link( $args );
+			wp_redirect( $redirect_url );
+			exit;
+		}
+
 		/**
-		 * Form validation for editing Usergroup
-		 */	
-		// Check if name field was filled in
-		if ( empty( $name ) ) {
-			$change_error = new WP_Error( 'invalid', __( 'Please enter a name for the user group.', 'edit-flow' ) );
-			die( $change_error->get_error_message() );
+		 * Handles a request to delete a Usergroup.
+		 * Hooked into 'admin_init' and kicks out right away if no action
+		 *
+		 * @since 0.7
+		 */
+		public function handle_delete_usergroup() {
+			if ( ! isset( $_GET['page'], $_GET['action'], $_GET['usergroup-id'] )
+			|| $_GET['page'] != $this->module->settings_slug || 'delete-usergroup' != $_GET['action'] ) {
+				return;
+			}
+
+			if ( ! isset( $_GET['nonce'] ) || ! wp_verify_nonce( $_GET['nonce'], 'delete-usergroup' ) ) {
+				wp_die( esc_html( $this->module->messages['nonce-failed'] ) );
+			}
+
+			if ( ! current_user_can( $this->manage_usergroups_cap ) ) {
+				wp_die( esc_html( $this->module->messages['invalid-permissions'] ) );
+			}
+
+			$result = $this->delete_usergroup( (int) $_GET['usergroup-id'] );
+			if ( ! $result || is_wp_error( $result ) ) {
+				wp_die( esc_html__( 'Error deleting user group.', 'edit-flow' ) );
+			}
+
+			$redirect_url = $this->get_link( array( 'message' => 'usergroup-deleted' ) );
+			wp_redirect( $redirect_url );
+			exit;
 		}
-		// Check that the name doesn't exceed 40 chars
-		if ( strlen( $name ) > 40 ) {
-			$change_error = new WP_Error( 'invalid', __( 'User group name cannot exceed 40 characters. Please try a shorter name.' ) );
-			die( $change_error->get_error_message() );
+
+		/**
+		 * Handle the request to update a given Usergroup via inline edit
+		 *
+		 * @since 0.7
+		 */
+		public function handle_ajax_inline_save_usergroup() {
+
+			if ( ! isset( $_POST['inline_edit'] ) || ! wp_verify_nonce( $_POST['inline_edit'], 'usergroups-inline-edit-nonce' ) ) {
+				die( esc_html( $this->module->messages['nonce-failed'] ) );
+			}
+
+			if ( ! current_user_can( $this->manage_usergroups_cap ) ) {
+				die( esc_html( $this->module->messages['invalid-permissions'] ) );
+			}
+
+			$usergroup_id = isset( $_POST['usergroup_id'] ) ? (int) $_POST['usergroup_id'] : 0;
+			if ( ! $existing_term = $this->get_usergroup_by( 'id', $usergroup_id ) ) {
+				die( esc_html( $this->module->messages['usergroup-missing'] ) );
+			}
+
+			$name = isset( $_POST['name'] ) ? sanitize_text_field( trim( $_POST['name'] ) ) : '';
+			$description = isset( $_POST['description'] ) ? stripslashes( wp_filter_nohtml_kses( trim( $_POST['description'] ) ) ) : '';
+
+			/**
+			 * Form validation for editing Usergroup
+			 */
+			// Check if name field was filled in
+			if ( empty( $name ) ) {
+				$change_error = new WP_Error( 'invalid', esc_html__( 'Please enter a name for the user group.', 'edit-flow' ) );
```

Comment:
```suggestion
				$change_error = new WP_Error( 'invalid', __( 'Please enter a name for the user group.', 'edit-flow' ) );
```

Reply:
This one's like this for the same reason as the other modules. The linter forces it and I'd rather not have ignores all over the place